bogota-july-2018/JSCalculator/assets/js/script.js

```diff
@@ -0,0 +1,81 @@
+
+var operations = {
+	"+": (num1, num2) => num1 + num2,
+	"-": (num1, num2) => num1 - num2,
+	"*": (num1, num2) => num1 * num2,
+	"/": (num1, num2) => num2 !== 0 ? num1/num2 : "Error!, use Clear button",
+	"":(num1, num2) => num2
+}
+
+var keyCodes = {
+	"106": "*",
+	"107": "+",
+	"109": "-",
+	"111": "/"
+}
+
+var res = 0;
+var first = true;
+var equal = false;
+var opc = "";
+var num2 = 0;
+
+var input = document.querySelector("#input-numbers");
+var output = document.querySelector("output");
```

Comment:
Is NOT  a good idea to have this kind of selectors, if some other dev is NOT aware of this code, he can add another `output` breaking your code, please use a class.